### ISSUE TITLE
Centos 7 repo is broken, clashes with EPEL's more recent zeromq version

### ISSUE THREAD

**Issue by sivann** [saltstack/salt#34629]
### Description of Issue/Question
```
Error: Package: python-zmq-14.7.0-1.el7.x86_64 (repo-saltstack-el7)
           Requires: libzmq.so.4()(64bit)
           Available: zeromq-4.0.5-4.el7.x86_64 (repo-saltstack-el7)
               libzmq.so.4()(64bit)
           Installed: zeromq-4.1.4-5.el7.x86_64 (@epel)
              ~libzmq.so.5()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

The only way to install salt is installing the 2015.10 version from EPEL, by disabling salt's repo:
```
[root@vserver-dev-4.gr1.inaccess.com:/src]$ yum install salt-minion --disablerepo=repo-saltstack-el7

```

It's better to release salt on EPEL more frequently than trying to maintain a separate repo. In the current state the systems cannot update without manual intervention.

**Comment by rallytime** [comment]
Thanks @sivann - This looks to me like a duplicate of #34575. I'm going to close this in favor of the older issue so we can track this in one place.
